### PR TITLE
Update full-node-binaries.md

### PR DESCRIPTION
**Pull request by HarshitDLT**
1. Instead of **sudo service bor start** , run **bash start.sh** inside _cd ~/node/bor_. Because start command is not present for bor.
2. In **make bor-all** command, **bor-all** will no longer supported so this should be **_make bor_**

### CODE REVIEW

**Comment by ashucoder9** [comment]
Thanks @HarshitDLT, we are forwarding this for SME review before pushing to Wiki.

File: docs/develop/network-details/full-node-binaries.md [hunk 1]
```diff
@@ -118,7 +118,7 @@ cd bor
 # Checkout to a proper version
 # For e.g: git checkout 0.3.3
 git checkout <TAG OR BRANCH>
-make bor-all
+make bor
```
Comment (0xKrishna) [inline docs/develop/network-details/full-node-binaries.md:121]: This is correct

File: docs/develop/network-details/full-node-binaries.md [hunk 2]
```diff
@@ -252,7 +252,7 @@ Run the full Heimdall node with these commands on your Sentry Node:
 
 ```bash
 sudo service heimdalld start
-sudo service heimdalld-rest-server start
+sudo service heimdalld rest-server start
```
Comment (0xKrishna) [inline docs/develop/network-details/full-node-binaries.md:255]: This is incorrect, In latest version we don't have separate rest-server anyway, Remove this change, We will raise another PR to fix this soon.

**Comment by cerberushades** [comment]
@HarshitDLT can you make the necessary changes according to @0xKrishna 's comments?

**Pull request closed without merging**